test/integration/federation/crud_test.go

```diff
@@ -23,21 +23,23 @@ import (
 	"github.com/pborman/uuid"
 
 	pkgruntime "k8s.io/apimachinery/pkg/runtime"
+	federationapi "k8s.io/kubernetes/federation/apis/federation/v1beta1"
 	"k8s.io/kubernetes/federation/pkg/federatedtypes"
 	"k8s.io/kubernetes/federation/pkg/federatedtypes/crudtester"
 	"k8s.io/kubernetes/test/integration/federation/framework"
 )
 
 // TestFederationCRUD validates create/read/update/delete operations for federated resource types.
 func TestFederationCRUD(t *testing.T) {
+	objectExpected := true
```

Comment:
I think I see what you're trying to do here - provide a symbolic value for whether an object is expected - but I don't think a variable is a good way to accomplish.  Even a constant would be a strange way to implement it, since you're proposing to negate the value in some of its calls.  I think it would be preferable to provide a call-specific boolean, either as a literal or value defined right before the call:

```golang
// literal
crudTester.CheckPropagationForClients(..., true)

// call-specific variable
objectExpected := true
crudTester.CheckPropagationForClients(..., objectExpected)
```